packages/common/src/models/Wallet.ts

```diff
@@ -4,8 +4,10 @@ import { Brand } from 'utils/typeUtils'
 
 export type StringWei = Brand<string, 'stringWEI'>
 export type StringAudio = string
+export type StringUSDC = string
```

Comment:
I updated this one to be branded. Since we use `StringAudio` in a lot of places, I don't want to change it for now in case there's som unknown side effect.